environs/config/config.go

```diff
@@ -2094,6 +2106,11 @@ data of the store. (default false)`,
 		Type:  environschema.Tbool,
 		Group: environschema.EnvironGroup,
 	},
+	DisableTelemetryKey: {
+		Description: `Whether the model disables telemetry`,
```

Comment:
Can we add maybe a link to some documentation about what we're reporting here?

```suggestion
		Description: `Disable telemetry reporting model information`,
```